.github/workflows/staleness_check.yml

```diff
@@ -20,15 +25,20 @@ jobs:
       matrix:
         branch: [main, 25.x, 27.x, 28.x]
         os: [{ name: Linux, value: ubuntu-latest}]
+        exclude:
+          # If we are in a presubmit run, only test main
+          - branch: ${{ !inputs.continuous-run && '25.x' }}
```

Comment:
Can we flip this to make it more maintainable?

Reply:
That's what I originally wanted, but the include syntax unfortunately doesn't work the same as the exclude so I can't have a condition like this. All the other options I considered seemed more confusing than this one, but I'm definitely open to other ideas!

Reply:
What if we removed `main` from the `branch` list and just exclude *all* of them on presubmits.  We should be able to add `main` back in with `include:` then to always run

Reply:
GHA doesn't support syntax to exclude all branches unless we explicitly list them, which leaves us in the same spot.